src/lib/Libifl/int_status.c

```diff
@@ -572,6 +579,13 @@ PBSD_status_aggregate(int c, int cmd, char *id, void *attrib, char *extend, int
 					cur = last;
 				}
 			}
+		} else if (!single_itr && (pbs_errno == PBSE_UNKQUE || pbs_errno == PBSE_UNKRESVID)) {
```

Comment:
Even in the said case, we are not masking PBSE_UNKQUE if it is non-reservation queue. This is because we are not clearing pbs_errno for all iterations.